client/src/components/JobList.vue

```diff
@@ -28,21 +33,25 @@
   import projectStatus from '@/utils/projectStatus';
   import jobsApi from '@/api/jobs';
   import AppHeader from '@/components/AppHeader';
+  import JobListFilters from '@/components/JobListFilters';
   import JobCard from '@/components/JobCard';
   import Circle from 'vue-loading-spinner/src/components/Circle';
+  import countries from '@/utils/countries';
 
   export default {
 
     name: 'JobList',
 
     components: {
       AppHeader,
+      JobListFilters,
       JobCard,
       'circle-loader': Circle,
     },
 
     data() {
       return {
+        allJobs: [],
         jobs: [],
```

Comment:
Now using two variables, one to store API answer, the other to hold the displayed jobs list.

Reply:
and jobsFromApi